benchmarks/README.md

```diff
@@ -81,3 +81,18 @@ These steps help reproduce the failures in TFLite models.
 5. Then compile the input MLIR file with `iree-compile`. The exact flags used
    to compile and run the benchmarks can be found in
    [this CMakeLists.txt file](./TFLite/CMakeLists.txt).
+
+### Run benchmark suites locally
+
+First you need to have `iree-import-tflite` and `requests` in your python environment. Then you can build the target `iree-benchmark-suites` to generate the required files:
+```sh
+// Assume your IREE build directory is $IREE_BUILD_DIR.
+
+cmake --build $IREE_BUILD_DIR --target iree-benchmark-suites
+```
```

Comment:
You could also add a note about building `iree-benchmark-import-models` to just import the models (if you want to run custom benchmarks or do other work with the files, without building the full android/linux benchmark suites)

https://github.com/google/iree/blob/7eb87c3d0d6a3be4839add003958dee789b5d1a6/CMakeLists.txt#L581-L593